varcode/transcript_mutation_effects.py

```diff
@@ -313,7 +315,7 @@ def get_class(effect):
             return _MultipleSubstitution
     return effect.__class__
 
-variant_effect_priority_list = [
+transcript_effect_priority_list = [
```

Comment:
if you go the latter route, I'd suggest implementing that function in the parent class (TranscriptMutationEffect), so all variant effects have a .priority (int) property.